lib/impress.client.js

```diff
@@ -787,26 +793,27 @@ Client.prototype.runScript = function(
   const application = client.application;
 
   application.createScript(fileName, (err, fn) => {
-    const fnWrapper = () => client.executeFunction(fn, callback);
     if (err) {
       client.err = err;
-      callback();
-    } else if (handler === 'access') {
+      return callback();
+    }
+    if (handler === 'access') {
       Object.assign(client.access, fn);
```

Comment:
If the function has some metadata that is being copied to `client.access`, I'd like it to be explicitly documented in a comment.